scripts/deployTest.ts

```diff
@@ -0,0 +1,417 @@
+import hre, {ethers, upgrades} from "hardhat";
+import {encodeProgramParameters, encodeRoundParameters} from "./utils";
+import {AddressZero} from "@ethersproject/constants";
+import {address} from "hardhat/internal/core/config/config-validation";
+
+const STATUS = {
+    PENDING: 0,
+    ACCEPTED: 1,
+    REJECTED: 2,
+    CANCELED: 3,
+};
```

Comment:
we can use an enum here

Reply:
I would strongly advise against using Typescript enums, for reasons nicely explained in this video: https://www.youtube.com/watch?v=0fTdCSH_QEU